### ISSUE TITLE
RouterWatchdog aggregate the heartbeat packets 

### ISSUE THREAD

**Issue by nialscorva** [ozone-development/ozp-iwc#200]
Have each RouterWatchdog aggregate the heartbeat packets for a peer into one, instead of each peer sending 7+ "set" packets to names.api every 10 seconds.

This would require the names.api to accept multiple heartbeat statuses in one request.  Perhaps by updating the router entry and making addresses a child of the router.

**Comment by Kevin-K** [comment]
PacketRouter greatly simplified  this.

Adding a `heartbeat` action for the names.api, the resource is trivial, but set to `/heart`.

The route expects the packet's entity to be an array of packets (participant.fixPacket()) and then distributes the packets accordingly using `routePacket`. Effectively this is a `bulkSend` action. Not made visible to the client API as it requires properly formatted packets. 

Typing that all out makes me want to revisit this and make it a default route that doesn't care about resource: 
```
participant.send({
    'dst': "names.api",
    'action': "bulkSend",
    'entity': [ {formattedPacket}, {formattedPacket}, ...] 
});
```

once a packet reaches its API's mutex holder, we don't care how many route packets are called. The bottleneck is transporting and this will cut down to 1 (or multiple if too large and needs fragmenting) packet.

**Comment by Kevin-K** [comment]
##Test 1: Packet transmition
1.) Open IWC debugger's traffic snooper:  http://localhost:13000/debugger.html#/traffic-snooper
2.) Set filter to watch all packets to/from "names.api": `data.dst === "names.api" || data.src === "names.api" `
3.) Enable the snooper.

**Passing criteria:**
* Packets of action `bulkSend` should have a **destination** of names.api. [A]
* Packets with response `ok` should have a **source** of names.api. [B]
* There should be a pattern of 1 bulkSend sent to the names.api followed by 1 ok response from the names.api and repeated every 10 seconds per open instance of the bus. ([A],[B],[A],[B],...)

##Test 2: names.api resource updating
1.) Open IWC debugger's  names.api tab:  http://localhost:13000/debugger.html#/names-api
2.) enter `/router` in the resource text input.
3.) wait 10 seconds for the resource to be set. Hit refresh all.
4.) Hit the watch icon (eye ball).

**Passing criteria:**
* The watched resource's entity property `time` should update every 10 seconds

**Comment by mparizernc** [comment]
Based upon the suggested tests verified this is happening on develop.

**Comment by Kevin-K** [comment]
Merged in PR #283, Testing verified.
